datumaro/plugins/imagenet_txt_format.py

```diff
@@ -81,10 +87,13 @@ def _load_items(self, path):
 
 class ImagenetTxtImporter(Importer):
```

Comment:
```suggestion
class ImagenetTxtImporter(Importer, CliPlugin):
    @classmethod
    def build_cmdline_parser(cls, **kwargs):
        parser = super().build_cmdline_parser(**kwargs)
        parser.add_argument('-l', '--labelmap', dest='labels_file',
            help="Path to the labelmap (synsets) file")

        return parser
```

Reply:
Why should the key be called `--labelmap` when the parameter is `labels_file`?

Reply:
Call it how you see fits better. Label map is kind of a general term for such things.

Reply:
I looked at existing occurrences of "label map" in the codebase, and it seems like that's usually used to describe maps of labels to colors (or vice versa). Whereas in this case we just have a list of labels, so I ended up going with `--labels_file`. I also didn't add a short option, since I don't expect this option to be frequently used.